src/store/actions/birdObservationsListActions.ts

```diff
@@ -0,0 +1,35 @@
+import { createAsyncAction, createStandardAction } from "typesafe-actions";
+import { BIRD_OBSERVATIONS_LIST_NAMESPACE } from "../../app/features/bird-info/conf";
+import {
+  BirdObservationData,
+  BirdObservationFilters,
+  VerificationStatus
+} from "../../app/features/bird-info/models";
+import { dataGridActions } from "../../components/table/dataGridActions";
+
+export const birdObservationsData = createAsyncAction(
+  "BIRD_OBSERVATIONS_REQUEST",
+  "BIRD_OBSERVATIONS_SUCCESS",
+  "BIRD_OBSERVATIONS_FAILURE"
+)<void, BirdObservationData[], string>();
+
+export const observationsFiltersRequest = createStandardAction(
+  "BIRD_OBSERVATIONS_FILTERS_REQUEST"
+)();
+
+export const setObservationVerificationStatus = createAsyncAction(
```

Comment:
i don't think the verification actions should be duplicated

Reply:
fixed